README.md

```diff
@@ -0,0 +1,16 @@
+# TheDailyAbstractionLambdas
+
+## Installation
+
+You'll need to install gradle in order to build the smithy model to generate the typescript client. 
+
+```
+brew install gradle
+```
+
+## Building the model
+
+Execute the following gradle command to generate all the necessary Smithy models and their OpenAPI definitions.
+```
+./gradlew build
```

Comment:
~I just realize that isn't it supposed to be `./gradle build`?~

You are using wrapper so `./gradlew build` command is right